### PR TITLE
Finish cookie cutter

### PR DESCRIPTION
**Pull request by nntai**
Ref N/A

Create a cookie cutter template for react native with expo app

### CODE REVIEW

**Review by ltankey — CHANGES_REQUESTED** [review]
Big effort, lots of changes. Most things wrong with this will probably be caught once we're using it.

We need to include `jest-junit-reporter` and related `package.json` stuff so that our CI test results can be interpreted by Jenkins.

An example from TSIS of the `package.json` config:

```
  "jest": {
    "preset": "jest-expo",
    "transformIgnorePatterns": [
      "node_modules/(?!react-native|react-navigation)/"
    ],
    "testEnvironment": "jest-environment-jsdom-global",
    "setupTestFrameworkScriptFile": "./setupTests.js",
    "testResultsProcessor": "./node_modules/jest-junit-reporter"
  }
```

**Comment by monkeypants** [comment]
Should we merge this as-is and make a new ticket for the jest-junit-reporter (to unblock #3), or get this right before merging in?

**Review by ltankey — APPROVED** [review]

**Pull request merged**
